src/components/Table/index.jsx

```diff
@@ -25,7 +31,7 @@ const Table = ({ data }) => {
               <td>{age}</td>
               <td>
                 <button
```

Comment:
good job but be consider that 

- [ ]  delete button should be the last button by  best user experiance